dev/tests/static/framework/Magento/Sniffs/Annotation/ClassAnnotationStructureSniff.php

```diff
@@ -97,8 +97,12 @@ public function process(File $phpcsFile, $stackPtr)
     {
         $tokens = $phpcsFile->getTokens();
         $previousCommentClosePtr = $phpcsFile->findPrevious(T_DOC_COMMENT_CLOSE_TAG, $stackPtr - 1, 0);
-        $this->validateAnnotationBlockExists($phpcsFile, (int)$previousCommentClosePtr, (int)$stackPtr);
-        $commentStartPtr = (int)$phpcsFile->findPrevious(T_DOC_COMMENT_OPEN_TAG, $stackPtr - 1, 0);
+        if (!$previousCommentClosePtr) {
+            $phpcsFile->addError('Comment block is missing', $stackPtr -1, 'MethodArguments');
+            return;
+        }
+        $this->validateAnnotationBlockExists($phpcsFile, $previousCommentClosePtr, (int)$stackPtr);
```

Comment:
Is this cast to `(int)` still necessary?

Reply:
@orlangur Nope,  can i remove it ?